test/e2e/mcd_test.go

```diff
@@ -153,6 +154,128 @@ func TestKernelArguments(t *testing.T) {
 	require.Nil(t, err)
 }
 
+func TestNoReboot(t *testing.T) {
+	cs := framework.NewClientSet("")
+	unlabelFunc := labelRandomNodeFromPool(t, cs, "worker", "node-role.kubernetes.io/infra")
+	oldInfraRenderedConfig := getMcName(t, cs, "infra")
+
+	infraNode := getSingleNodeByRole(t, cs, "infra")
+
+	output := execCmdOnNode(t, cs, infraNode, "cat", "/rootfs/proc/uptime")
+	oldTime := strings.Split(output, " ")[0]
```

Comment:
Looking at the test failure are you sure this oldTime is correct? maybe it should be a different index?

Reply:
command will return correct value, but it seems to go into api timeout  error when we call `cat /rootfs/uptime` through execCmdOnNode() . See the error message in https://github.com/openshift/machine-config-operator/pull/2259#issuecomment-736298449 from one of the previous gcp-op run on this [job](https://prow.ci.openshift.org/view/gs/origin-ci-test/pr-logs/pull/openshift_machine-config-operator/2259/pull-ci-openshift-machine-config-operator-master-e2e-gcp-op/1333453313349259264)

Reply:
While replying to you, I understand what is happening. execCmdOnNode() combines both stdout and error and returns it which is causing the problem.